components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/DiscoverableGroup.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Add a description for the validation function and the utility method for building discoverable groups.

Reply:
addressed by https://github.com/wso2/carbon-identity-framework/pull/6283/commits/7a616cd702ac969d0adb14ed262c88cc7805bfe3